<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>template_device</class>
 <widget class="QWidget" name="template_device">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>280</width>
       <height>210</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>280</width>
       <height>210</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <widget class="QPushButton" name="onnetworkPair_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Onnetwork pair</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BLE_pair_btn">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>120</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>BLE pairing</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Device ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Device name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Device type</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_deviceID">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_DeviceName">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>70</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_DeviceType">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="control_btn">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>control</string>
      </property>
     </widget>
     <widget class="QPushButton" name="info_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>More info</string>
      </property>
     </widget>
     <widget class="QPushButton" name="unpair_btn">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>120</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>unpair</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
